### DISCUSSION TITLE
Add torlook.info tracker

### DISCUSSION THREAD

**Discussion by VuzzyM** [KiraLT/torrent-stream-server#266] [Ideas]
Add torlook.info tracker and add authorization on server, user and password for vps.

**Comment by KiraLT** [comment]
Hello,

torlook.info looks great, I will check if we can integrate it.

As for authorization, you can use API key as a password - [check docs](https://github.com/KiraLT/torrent-stream-server/wiki/Security).

**Comment by VuzzyM** [comment]
@KiraLT authorization implement to web panel settings, no api.

**Comment by KiraLT** [comment]
It protects both API and Web user interface with the same key. You will be required to enter API key when you open the website:

![authorization](https://user-images.githubusercontent.com/10944908/124577049-96e35780-de55-11eb-8700-7a5aad99e643.png)

**Comment by brkcoc02** [comment]
Hi @KiraLT , if possible please also add rutracker.org rarbg.to too.

**Comment by KiraLT** [comment]
I am working on torrents search rework. [torlook.info](torlook.info) is a nice service, however, it is not possible to filter search results by category and it does not return all information (such as IMDB id which we can use to retrieve subtitles- #264). So I thought to crawl multiple providers at the same time, so we would achieve the same result with more functionality. You can follow the progress on #265.
